src/components/Markdown/MarkdownLink.tsx

```diff
@@ -26,6 +30,36 @@ const scrollWithTimeout = (element: any, offset: number) => {
   }, 400);
 };
 
+const CanEmbed: React.FC<{ href: string }> = ({ href }) => {
+  const [isOpen, setIsOpen] = useState(false);
+  const region =
+    href === 'https://covidactnow.org/embed/us/fips/19100'
+      ? regions.findByFipsCode('19100')!
+      : undefined;
+  return (
+    <div>
+      <CenterEmbed>
+        <iframe
+          src={href}
+          title="Covid Act Now"
+          width="400"
+          height="390"
+          frameBorder="0"
+          scrolling="no"
+        />
+        <EmbedPrompt>
+          <span onClick={() => setIsOpen(true)}>Embed on your website</span>
+        </EmbedPrompt>
+      </CenterEmbed>
+      <EmbedPreview
+        open={isOpen}
+        onClose={() => setIsOpen(false)}
+        region={region}
+      />
```

Comment:
Although the link is nested under `<CenterEmbed>`, the second link does not appear perfectly centered. I think this is due to the size of the location specific embed preview box directly above it.